### DISCUSSION TITLE
Consider integrating any-sync or similar technology

### DISCUSSION THREAD

**Discussion by dphaldes** [AppFlowy-IO/AppFlowy#3559] [Ideas]
[Any-sync](https://github.com/anyproto/any-sync) is an open source source for peer to peer syncing. From the repo description, it  is encrypted and local-first and makes it ideal for appflowy! Do consider it when you get into  adding syncing functionality in the future! :)

**Comment by trymeouteh** [comment]
Or even an option to choose a directory to store notes in and allow users to use Syncthing to sync notes

https://forum.appflowy.io/t/sync-to-device-folder-of-your-choosing/932

**Comment by Fau57** [comment]
Syncthing 4theWin
